generated/nidaqmx/constants.py

```diff
@@ -68,8 +68,8 @@ class Action(Enum):
 
 
 class ActiveLevel(Enum):
-    ABOVE = 10093  #: Pause the measurement or generation while the signal is above the threshold.
-    BELOW = 10107  #: Pause the measurement or generation while the signal is below the threshold.
+    ABOVE_LVL = 10093  #: Pause the measurement or generation while the signal is above the threshold.
+    BELOW_LVL = 10107  #: Pause the measurement or generation while the signal is below the threshold.
```

Comment:
This change I have less of a problem with. However, I think we should avoid shorthand versions of words in Python. So LVL should be LEVEL, I think.

Reply:
Updated enum values as in expected constants.py (ABOVE,BELOW)